rootfs/scheduler/k8s.py

```diff
@@ -503,35 +535,51 @@ def _get_service(self, name, namespace):
 
         return response
 
+    def _create_secret(self, namespace):
```

Comment:
IMO the function name is too generic if it is `minio` secrets only; can be changed later tho